### PR TITLE
Added local-storage related functions in utility module properly

### PR DESCRIPTION
**Pull request by anubhav-here**
The idea is to segregate all the local-storage related functions in a proper module.
This PR aims to fix that.
Signed-off-by: Anubhav Deep <adeep@redhat.com>

### CODE REVIEW

File: ocs_ci/utility/localstorage.py [hunk 1]
```diff
@@ -32,3 +38,122 @@ def fetch_all_device_paths():
...
+    cur_device_list = output["spec"]["storageClassDevices"][0]["devicePaths"]
+    path = os.path.join(constants.EXTERNAL_DIR, "device-by-id-ocp")
+    utils.clone_repo(constants.OCP_QE_DEVICEPATH_REPO, path)
+    os.chdir(path)
+    utils.run_cmd("ansible-playbook devices_by_id.yml")
+    with open("local-storage-block.yaml", "r") as cloned_file:
+        with open("local-block.yaml", "w") as our_file:
+            device_from_worker1 = device_sets_required
+            device_from_worker2 = device_sets_required
+            device_from_worker3 = device_sets_required
```
Comment (ebenahar) [inline ocs_ci/utility/localstorage.py:68]: Please use a dynamic arg for the vars representing the workers. You can use config.ENV_DATA['worker_replicas']
for that

Comment (anubhav-here) [reply ocs_ci/utility/localstorage.py:68]: Will fix that!

**Comment by anubhav-here** [comment]
@clacroix12 @ebenahar So, rather than assuming 3 worker nodes everytime, used the approach you guys suggested.

File: ocs_ci/utility/localstorage.py [hunk 2]
```diff
@@ -32,3 +39,114 @@ def fetch_all_device_paths():
...
+
+    Args:
+        device_sets_required (int) : Count of device sets to be added
+        osd_size_capacity_requested (int) : Requested OSD size capacity
+
+    Returns:
+        list : List containing added device paths
+
+    """
+    ocp_obj = OCP()
```
Comment (ebenahar) [inline ocs_ci/utility/localstorage.py:56]: please instantiate OCP() with `OCP(kind='localvolume')`.
Then, in line 60, you will be able to do `oc_obj.get()` instead

File: ocs_ci/utility/localstorage.py [hunk 3]
```diff
@@ -32,3 +39,114 @@ def fetch_all_device_paths():
...
+    workers = get_typed_nodes(node_type="worker")
```
Comment (ebenahar) [inline ocs_ci/utility/localstorage.py:57]: general note - please add more comments in the code. It is hard to follow everything that is going on

Comment (anubhav-here) [reply ocs_ci/utility/localstorage.py:57]: Added!

File: ocs_ci/utility/localstorage.py (hunk 2 above)
Comment (anubhav-here) [reply ocs_ci/utility/localstorage.py]: Fixed!

File: ocs_ci/utility/localstorage.py (hunk 3 above)
Comment (ebenahar) [reply ocs_ci/utility/localstorage.py:57]: thanks!
